code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -299,13 +299,13 @@
 		..()
 		return
 	if (!cover_open)
-		to_chat(user, span_warning("[src]'s cover is closed! Open it before trying to remove the magazine!"))
+		balloon_alert(user, "cover must be opened first!")
```

Comment:
Change these to "open the cover!", which is shorter and more direct (and more common with balloons)